docs/Operators.md

```diff
@@ -77,11 +77,12 @@ All the specific operators are listed below, each featured with several capabili
 | alphanumeric_filter            | General | en, zh | Keeps samples with alphanumeric ratio within the specified range                           |
 | average_line_length_filter     | Code    | en, zh | Keeps samples with average line length within the specified range                          |
 | character_repetition_filter    | General | en, zh | Keeps samples with char-level n-gram repetition ratio within the specified range           |
-| clip_similarity_filter      | Multimodal |   -    |  Keeps samples with similarity between text and images within the specified range           |
 | flagged_words_filter           | General | en, zh | Keeps samples with flagged-word ratio below the specified threshold                        |
 | image_aspect_ratio_filter      | Image   |   -    | Keeps samples contains images with aspect ratios within specific range                     |
 | image_shape_filter             | Image   |   -    | Keeps samples contains images with widths and heights within specific ranges               |
-| image_size_filter      | Image   |   -    | Keeps samples contains images whose size in bytes are within specific range                     |
+| image_size_filter              | Image   |   -    | Keeps samples contains images whose size in bytes are within specific range                     |
+| image_text_matching_filter     | Multimodal |   -    |  Keeps samples with matching score between text and images within the specified range           |
+| image_text_similarity_filter   | Multimodal |   -    |  Keeps samples with similarity between text and images within the specified range           |
```

Comment:
I think it's better to show that "matching score" is from blip and "similarity" is from "clip". Otherwise, they are too similar to tell them apart.